setup.py

```diff
@@ -87,7 +87,7 @@
         'websocket-client',
         'yaspin',
         # KFP runtime dependencies
-        'kfp>=1.6.3<1.7',
+        'kfp~=1.6.3',
         'kfp-tekton==0.8.1',
```

Comment:
Should we do the same with `kfp-tekton`? (`~=0.8.1`). That's the only remaining `==` requirement we have.